cli/githubutil.py

```diff
@@ -140,3 +141,101 @@ def release_note_blocks_cli(
         repository_branch=repository_branch,
         commit_range=commit_range
     ).release_note_blocks()
+
+
+def cleanup_draft_releases(
+    github_cfg_name: str,
+    github_repository_owner: str,
+    github_repository_name: str,
+    latest_release_version: str,
+    dry_run: bool=True,
+):
+    github_cfg = ctx().cfg_factory().github(github_cfg_name)
+
+    githubrepobranch = GitHubRepoBranch(
+        github_config=github_cfg,
+        repo_owner=github_repository_owner,
+        repo_name=github_repository_name,
+        branch='master',
+    )
+
+    helper = GitHubRepositoryHelper.from_githubrepobranch(
+        githubrepobranch=githubrepobranch,
+    )
+    # prepare semver for later comparison
+    latest_semver = semver.parse_version_info(latest_release_version)
+
+    # we're only interested in releases whose name can be parsed using semver and whose semver
+    # contains the prerelease-label 'draft'.
+    def _is_semver_parseable(release):
+        try:
+            release_semver = semver.parse_version_info(release.name)
+        except Exception:
+            return False
+        if release_semver.prerelease != 'draft':
```

Comment:
this is bad! the function name somewhat implies it is only checked whether the given str is a valid semver. This is a different semantics. Do not mix this. Or if you do mix, name the function differently